io/io/src/TFileMerger.cxx

```diff
@@ -387,6 +388,381 @@ Long64_t MergeRNTuples(TClass* rntupleHandle, const TString& /* target */, const
 
 } // anonymous namespace
 
+Bool_t TFileMerger::MergeOne(TDirectory *target, TList *sourcelist, Int_t type, TFileMergeInfo &info,
```

Comment:
I have not reviewed this function; I suppose it comes 1:1 from `MergeRecursive` through code refactoring?